sklearn/metrics/classification.py

```diff
@@ -1558,6 +1558,11 @@ def log_loss(y_true, y_pred, eps=1e-15, normalize=True, sample_weight=None):
         Predicted probabilities, as returned by a classifier's
         predict_proba method.
 
+    labels : array-like 
+        When len(unique(y_true)) < len(unique(y_pred)), you must use labels option and
```

Comment:
WDYT about rewording this as , "if `labels` is not provided, the labels are inferred from `y_true` or more specifically, set to np.unique(y_true)"